### PR TITLE
Logistic regression

### PR DESCRIPTION
**Pull request by wnsgur4322**
small errors on img classification are fixed.
but confusion matrix library need to fix for working with multi-class classification (ex. iris data, MNIST data)

### CODE REVIEW

**Review by laurajjiang — APPROVED** [review]

**Comment by thuyvyng** [comment]
Screen Shot 2021-04-12 at 4 13 26 PM
The confusion matrix in logistic regression doesnt work in here & i believe the vector visualization should be included in this chapter (logisitic_regression_keras)

**Comment by thuyvyng** [comment]
Screen Shot 2021-04-12 at 4 15 22 PM
neural network classification's confusion matrix doesnt work either

**Review by thuyvyng — CHANGES_REQUESTED** [review]
confusion matrices are included but dont work, we need to fix/remove them. and i believe the logistic regression is supposed to have the vector visualization and i believe the barcode visualization should be included in a chapter as well

**Comment by thuyvyng** [comment]
For the intent of the PR, you can just remove all the calls to the confusion matrix and we can add them back in later or something

**Comment by MarkleyO** [comment]
@Thuy-Vy - I went through all the errors calling the Confusion Matrix on the linear regression keras notebook, and the neural network classification notebook. I've got a pretty strong grip on the issue here, the vizualization 'works' but the issue is that on the first failure, the JSON file was not being properly formatted, giving two different formats of data for the 'prediction' and 'label' fields. The issue on the second is similar, in that it is not formatted properly (epoch data needs to be encoded into the JSON, even if the user only wants to display a single epoch) and the proper formatting for data within that epoch was not used. 

However, I think you're right, go ahead and delete them from the PR for now, and push. The work that I'm already doing, which is setting up a module in the library for the extraction of data from the model will properly handle this once its complete, and have the visualizations working fine. However this is on a separate branch right now and will need to be incorporated a bit later

**Comment by thuyvyng** [comment]
> @Thuy-Vy - I went through all the errors calling the Confusion Matrix on the linear regression keras notebook, and the neural network classification notebook. I've got a pretty strong grip on the issue here, the vizualization 'works' but the issue is that on the first failure, the JSON file was not being properly formatted, giving two different formats of data for the 'prediction' and 'label' fields. The issue on the second is similar, in that it is not formatted properly (epoch data needs to be encoded into the JSON, even if the user only wants to display a single epoch) and the proper formatting for data within that epoch was not used.
> 
> However, I think you're right, go ahead and delete them from the PR for now, and push. The work that I'm already doing, which is setting up a module in the library for the extraction of data from the model will properly handle this once its complete, and have the visualizations working fine. However this is on a separate branch right now and will need to be incorporated a bit later

@MarkleyO when is ur pr gonna be up for the fixes, also as a side note i think you have to do my username @thuyvyng to notify me
